spec/acceptance/golden_masters_spec.rb

```diff
@@ -19,7 +19,7 @@
     end
   end
 
-  context 'with kafka and sequel' do
+  context 'with kafka and sequel', :kafka do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.